apps/website/src/content/docs/kbd.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Kbd
+description: This component is used to display which key or combination of keys performs a given action.
+thumbnail: Keyboard
```

Comment:
should we also rename the thumbnail?

Reply:
no need to touch ladle or anything, it's fine to leave as-is

for the thumbnail, you also need to rename the actual thumbnail inside `website/src/icons/thumbnails`